helmctlr/helm.go

```diff
@@ -53,15 +53,15 @@ func NewController(chartDir, ns, rn, host string, wait bool, waitto int64, logge
 		ns = defaultNS
 	}
 	c := &Controller{
-		Helm:        helm.NewClient(helm.Host(host)),
-		ChartDir:    chartDir,
-		Namespace:   ns,
-		ReleaseName: rn,
-		Wait:        wait,
-		WaitTimeout: waitto,
-		dynClient:   dynClient,
-		kubeClient:  kubeClient,
-		logger:      logger,
+		Helm:           helm.NewClient(helm.Host(host)),
+		ChartDir:       chartDir,
+		Namespace:      ns,
+		ReleaseName:    rn,
+		Wait:           wait,
+		WaitTimeout:    waitto,
+		resourceClient: resourceClient,
+		kubeClient:     kubeClient,
```

Comment:
Remove `kubeClient`, it is not used or needed.

Reply:
we'll need kubeclient for the work Chance is about to do to write to configmaps, unless I misunderstood

Reply:
Then we should pass the `ResourceInterface` for `ConfigMaps`, not the full k8s client.

Reply:
The `fake` client works on this interface, so I'd prefer to keep it as it is.